code/modules/mob/living/simple_animal/hostile/headcrab.dm

```diff
@@ -22,7 +22,7 @@
 	ventcrawler = VENTCRAWLER_ALWAYS
 	var/datum/mind/origin
 	var/egg_lain = 0
-	gold_core_spawnable = HOSTILE_SPAWN //yogs
+	gold_core_spawnable = NO_SPAWN //yogs
```

Comment:
This leaves a ton of unused code in the codebase, get rid of all the xenoling framework.

Reply:
It's fine dude, jesus